### ISSUE TITLE
Add `# nocov start` and `# nocov end` comments before/after staticimports.R

### ISSUE THREAD

**Issue by elipousson** [wch/staticimports#25]
I have a few staticimport functions where I'm not using all features of the imported function–but also don't want to add tests to my package that are only there to test a static import function.

If staticimports could add a `# nocov start` before the imported functions and `# nocov end` after the imported function (similar to `use_standalone()`) that would be helpful.

**Comment by wch** [comment]
What do you think about adding a a `prefix` and `suffix` argument to `import()`? There are other directives that someone might want to add (like `# styler: on` and `# nolint start`) and I don't think it's the job of staticimports to dictate which ones are added and which ones aren't.

**Comment by elipousson** [comment]
@wch No, that approach makes a lot of sense. I’d be happy to try to put together a pull request if it is helpful.
